api/src/components/reference/controller.ts

```diff
@@ -1,6 +1,7 @@
 import * as I from 'interface';
 import * as response from 'lib/response';
 import * as referenceService from 'reference/service';
+import * as publicationService from 'publication/service';
 
 export const get = async (event: I.AuthenticatedAPIRequest<undefined, undefined, I.CreateReferencePath>) => {
```

Comment:
we should call this getAll because it brings all the references for a specific publication